File: sample/src/main/java/org/polaric/colorful/sample/SampleApp.java
package org.polaric.colorful.sample;

import android.app.Application;
import android.graphics.Color;

import org.polaric.colorful.Colorful;

public class SampleApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Colorful.defaults()
                .primaryColor(Colorful.ThemeColor.RED)
                .accentColor(Colorful.ThemeColor.BLUE)
                .translucent(false)
                .dark(true);

        Colorful.init(this);
    }
}

File: sample/src/main/java/io/multimoon/colorfulsample/SampleApp.kt
package io.multimoon.colorfulsample

import android.app.Application
import io.multimoon.colorful.Defaults
import io.multimoon.colorful.ThemeColor
import io.multimoon.colorful.initColorful

class SampleApp:Application() {
    override fun onCreate() {
        super.onCreate()
        initColorful(this, Defaults(ThemeColor.GREEN, ThemeColor.BLUE, false))

    }
}